fossunited/fossunited/forms.py

```diff
@@ -98,6 +126,11 @@ def unpublish_form(doctype, docname):
     Used to Unpublish RSVP and CFP forms by the user.
     This bypasses the permissions required to unpublish the form.
     """
+    if not is_valid_doctype(doctype):
+        frappe.log("Unauthorized usage of unpublish_form")
+        frappe.throw("Not allowed")
+        return
+
```

Comment:
Lots of repetition of this same set of lines, hinting there may be a better and DRY way.